src/golang/lib/workflow/operator/base.go

```diff
@@ -45,6 +45,7 @@ type baseOperator struct {
 	// This cannot be set if the operator is cache-aware, since this only happens in non-preview paths.
 	resultsPersisted bool
 	execMode         ExecutionMode
+	launchNumber     int
```

Comment:
What is launchNumber used for? If it's a Lambda specific thing can you add a comment